orderbook/src/api/post_quote.rs

```diff
@@ -354,7 +354,17 @@ pub fn post_quote(
         async move {
             let result = quoter.calculate_quote(&request).await;
             if let Err(err) = &result {
-                tracing::error!(?err, ?request, "post_quote error");
+                match err {
+                    OrderQuoteError::Fee(FeeError::PriceEstimate(PriceEstimationError::Other(
+                        err,
+                    ))) => {
+                        tracing::error!(?err, ?request, "post_quote Fee error");
+                    }
+                    OrderQuoteError::Order(ValidationError::Other(err)) => {
+                        tracing::error!(?err, ?request, "post_quote Order Validation error");
+                    }
+                    _ => tracing::warn!(?err, ?request, "post_quote User error"),
+                }
```

Comment:
Why not:
```suggestion
                tracing::warn!(?err, ?request, "post_quote error");
```

Are there any errors we want to alert on? From what I understand we can already API metrics to determine if we are failing to often. Specifically it should be: `gp_v2_api_api_requests_complete{exported_endpoint="post_quote",status_code="500"}` (pending https://github.com/gnosis/gp-v2-services/pull/1262/files).

Reply:
I did this because all the "Other" errors are our own internal server errors, for which I would assume we want to be alerted about.

Reply:
Changed it in favour of grafana.

Reply:
Any reason remaining that has prevented you from approving?

Reply:
Just wanted an answer to my question before approving :stuck_out_tongue:.